distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh

```diff
@@ -33,20 +34,21 @@ if [[ -z $JARFILE ]]; then
   exit 2
 fi
 
-if [[ $1 ]]; then
-    COMMONNAME="$1"
+if [[ $1 && $2 ]]; then
+    PARAM1="$1"
+    PARAM2="$2"
 else
-    COMMONNAME=$(hostname -f)
+    PARAM1="-cn"
+    PARAM2="$(hostname -f)"
 fi
 
 echo "--IGNORE SLF4J ERRORS"--
-
-$(java -Djavax.net.ssl.keyStore="$KEYFILE" -Djavax.net.ssl.keyStorePassword="$PASSWORD" -Djavax.net.ssl.keyStoreType="$KEYTYPE" -jar "$JARFILE" "$COMMONNAME")
+echo $PARAMS
```

Comment:
I tried running the shell and only a newline is echoed here... where should PARAMS be assigned a value?